src/components/home/index.css

```diff
@@ -0,0 +1,22 @@
+/* cards style */
+.cardsContainer {
+    background-color: rgb(221, 225, 235);
+  
+    padding: 10% 100px 10% 100px;
```

Comment:
can you include a screenshot of what you're making please? both the design and what it looks like

Reply:
this is what I've done.
<img width="1440" alt="my work" src="https://user-images.githubusercontent.com/58145973/88708322-2cda1280-d11c-11ea-9bf4-830907713ac8.png">
this is the design
<img width="880" alt="design" src="https://user-images.githubusercontent.com/58145973/88708490-61e66500-d11c-11ea-8233-4770742e2add.png">

Reply:
@louisrli here are the screenshots